server/sandbox.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
why is this called `sandbox.sh`? looks like it should be called `env.sh` or something...to me sandbox implies testing code, not production code